### DISCUSSION TITLE
账号续费后一直无法使用，要求退款

### DISCUSSION THREAD

**Discussion by wudaokou777** [getlantern/lantern-forum-cn#822] [General]
![Screenshot_20240831_092440_com eg android AlipayG](https://github.com/user-attachments/assets/8701d41a-0fd4-40a5-b0f4-b70f497fa715)
邮箱是924787857@qq.com

**Comment by MaxMKMax** [comment]
@wudaokou777 您好. 请问您需要退款的具体原因是什么呢? 
您要不再给我们一个机会? 希望您别退款, 可以继续用我们的服务. 如果那里我们做不好了, 我向您抱歉. 谢谢您支持蓝灯!

**Reply by wudaokou777** [reply]
那你就把验证码发给我，让我登录上去

**Comment by wudaokou777** [comment]
邮件收不到验证码，软件我没法使用啊，不是我想退，是服务都没有&nbsp;


栖迟
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Mike ***@***.***&gt;; 
发送时间: 2024年9月3日(星期二) 凌晨0:31
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [getlantern/lantern-forum-cn] 账号续费后一直无法使用，要求退款 (Discussion #822)














 
@wudaokou777 您好. 请问您需要退款的具体原因是什么呢?
 您要不再给我们一个机会? 希望您别退款, 可以继续用我们的服务. 如果那里我们做不好了, 我向您抱歉. 谢谢您支持蓝灯!
